src/sagemaker/serve/model_server/djl_serving/prepare.py

```diff
@@ -38,23 +38,39 @@ def _has_serving_properties_file(code_dir: Path) -> bool:
     return code_dir.joinpath(_SERVING_PROPERTIES_FILE).is_file()
 
 
-def _members(resources: object, depth: int):
+def _move_to_code_dir(js_model_dir: str, code_dir: Path):
     """Placeholder Docstring"""
-    for member in resources.getmembers():
-        member.path = member.path.split("/", depth)[-1]
-        yield member
+    js_model_resources = Path(js_model_dir).joinpath("model")
+    for resource in js_model_resources.glob("*"):
+        try:
+            shutil.move(resource, code_dir)
+        except shutil.Error as e:
+            if "already exists" in str(e):
```

Comment:
is there no chance the filepath will be the same but the content different?

Reply:
Yea there is a chance. We give the customer the ability to modify the contents of the model resources. In general we should not override. We are putting the responsibility on the customer to properly clean their directory if they want a fresh start.

Reply:
QQ here: Why are the DJL resources located in a model/ subfolder? Is this structure going to change?